service/src/test/java/bio/terra/catalog/service/BaseStatusServiceTest.java

```diff
@@ -23,4 +26,23 @@ void getCurrentStatus() {
         service.getCurrentStatus(),
         is(new SystemStatus().ok(true).systems(Map.of("test", status))));
   }
+
+  interface Status extends Supplier<SystemStatusSystems> {}
+
+  @Test
+  void startStatusChecking() {
+    var config = new StatusCheckConfiguration(true, 1, 0, 10);
+    BaseStatusService service = new BaseStatusService(config);
+    var status = mock(Status.class);
```

Comment:
can you use @mock here?

Reply:
You could, but you'd have to also annotate the class with `@ExtendWith(MockitoExtension.class)`. I think it's OK to call `mock()` directly since it's only used in one test rather than annotating the class. But either way would work.